pkg/reconciler/trigger/controller_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package trigger
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/logging"
+	"knative.dev/pkg/metrics"
+	. "knative.dev/pkg/reconciler/testing"
+	"knative.dev/pkg/system"
+	tracingconfig "knative.dev/pkg/tracing/config"
+
+	// Fake injection informers
+	_ "github.com/google/knative-gcp/pkg/client/injection/informers/broker/v1beta1/broker/fake"
+	_ "github.com/google/knative-gcp/pkg/client/injection/informers/broker/v1beta1/trigger/fake"
+	_ "knative.dev/pkg/client/injection/ducks/duck/v1/addressable/fake"
+	_ "knative.dev/pkg/client/injection/ducks/duck/v1/conditions/fake"
+	_ "knative.dev/pkg/client/injection/kube/informers/apps/v1/deployment/fake"
+	_ "knative.dev/pkg/client/injection/kube/informers/core/v1/configmap/fake"
+	_ "knative.dev/pkg/client/injection/kube/informers/core/v1/endpoints/fake"
+	_ "knative.dev/pkg/client/injection/kube/informers/core/v1/pod/fake"
+	_ "knative.dev/pkg/injection/clients/dynamicclient/fake"
+
+	// Why is serving needed here?
+	_ "knative.dev/serving/pkg/client/injection/informers/serving/v1/service/fake"
```

Comment:
What if you remove it?

Reply:
This comment is cargo-culted from similar tests in this repo. When the serving import is missing the test fails, but I don't remember what the error message is.

Reply:
Interesting, the error is gone if I delete it... I am pretty sure there was an error when I tried this last week.

I believe the error used to be 
```
--- FAIL: TestNew (0.00s)
    logger.go:130: 2020-05-28T10:29:15.343-0700 PANIC   service/service.go:48   Unable to fetch knative.dev/serving/pkg/client/informers/externalversions/serving/v1.ServiceInformer from context.
panic: Unable to fetch knative.dev/serving/pkg/client/informers/externalversions/serving/v1.ServiceInformer from context. [recovered]
        panic: Unable to fetch knative.dev/serving/pkg/client/informers/externalversions/serving/v1.ServiceInformer from context.
```

Going to delete this import wherever applicable. It seems that pkg/reconciler/intevents/topics still needs this.